src/main/java/core/basesyntax/processing/FruitTransaction.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.processing;
+
+import java.util.List;
+
+public interface FruitTransaction {
+
+    void processing(List<String> activities);
```

Comment:
You need to have the service and his impl with the name FruitTransactionParserService or kind like this with a method that gets List<String> that were read from the file and returns List<FruitTransaction>